lib/content-services/src/lib/document-list/components/document-list.component.ts

```diff
@@ -832,6 +836,10 @@ export class DocumentListComponent extends DataTableSchema implements OnInit, On
         this.columnsWidthChanged.emit(this.columnsWidths);
     }
 
+    onSelectedItemsCountChanged(count: number) {
```

Comment:
could you please write unit test inside unit tests for that component to test if this is function calls emit for selectedItemsCountChanged event?